detection-rules/body_job_scam_freemail_pivot.yml

```
@@ -23,18 +23,32 @@ source: |
       or not any(headers.hops, any(.fields, strings.ilike(.name, "In-Reply-To")))
     )
   )
-  and 3 of (
-    any([subject.subject, body.current_thread.text],
-        regex.icontains(., '(full|part).time')
-    ),
-    strings.ilike(body.current_thread.text, '*job*'),
-    regex.icontains(body.current_thread.text, '\bHR\b'),
-    strings.ilike(body.current_thread.text, '*manager*'),
-    strings.ilike(body.current_thread.text, '*commission*'),
-    strings.ilike(body.current_thread.text, '*hourly*'),
-    strings.ilike(body.current_thread.text, '*prior experience*'),
-    strings.ilike(body.current_thread.text, '*company rep*'),
-    strings.ilike(body.current_thread.text, "100% legal")
+  and (
+    3 of (
+      any([subject.subject, body.current_thread.text],
+          regex.icontains(., '(full|part).time')
+      ),
+      strings.ilike(body.current_thread.text, '*job*'),
+      regex.icontains(body.current_thread.text, '\bHR\b'),
+      strings.ilike(body.current_thread.text, '*manager*'),
+      strings.ilike(body.current_thread.text, '*commission*'),
+      strings.ilike(body.current_thread.text, '*hourly*'),
+      strings.ilike(body.current_thread.text, '*per hour*'),
+      strings.ilike(body.current_thread.text, '*prior experience*'),
+      strings.ilike(body.current_thread.text, '*company rep*'),
+      strings.ilike(body.current_thread.text, "100% legal")
+    )
+    or (
+      length(ml.nlu_classifier(body.current_thread.text).topics) == 1
+      and any(ml.nlu_classifier(body.current_thread.text).topics,
+              .name == "Professional and Career Development"
+              and .confidence == "high"
+      )
+      and (
+        length(recipients.to) == 0
+        or all(recipients.to, .display_name == "Undisclosed recipients")
```

catch where the display_name is `undisclosed-recipients` in additional to `undisclosed recipients`
```suggestion
        or all(recipients.to, strings.ilike(.display_name, "Undisclosed?recipients"))
```